mods/gennext/README.md

```diff
@@ -361,6 +363,8 @@ Minor learnset changes:
 
 - Masquerain gets Surf
 
+- Pidgeot gets Focus Blast
+
```

Comment:
Ugh, NEXT learnset changes are very rare and come with a lot more justification than this... I'm not sure what to think of this. Convince me.